extensions/amp-pinterest/0.1/amp-pinterest.js

```diff
@@ -350,7 +350,7 @@ class AmpPinterest extends AMP.BaseElement {
         '?pin_ids=' + pinId +
         '&sub=www&base_scheme=https';
       return call(query).then(r => {
```

Comment:
or change `renderPin` into an arrow function.

```
const renderPin = (r) => {

}

call(query).then(renderPin);
```

then we wouldn't need the `bind`.

i'm ok with either solution.